linux-packaging-appimage/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM --platform=linux/386 debian:bookworm-slim
```

Comment:
We usually prefer 64bit system with multiarch. This is done because opennox was shipping `noxtool`, which is 64bit. We may also ship a launcher at some point, which will also be 64bit.

Reply:
Docker assumes 64 bit multiarch when you specify 32 bit platform, because most hosts run 64 bit kernels and Docker shares the host kernel. Also noxtool is NOT included here, only opennox binaries which are 32 bit only at the moment. As soon as you switch to shipping 64 bit binaries, i'll adapt both script and Dockerfile accordingly.